CHANGELOG.md

```diff
@@ -42,7 +42,9 @@
 * [ENHANCEMENT] Rename batches property of Trace to ResourceSpans to be OTEL compatible [#3895](https://github.com/grafana/tempo/pull/3895)
 * [ENHANCEMENT] Reduce memory consumption of query-frontend[#3888](https://github.com/grafana/tempo/pull/3888) (@joe-elliott)
 * [ENHANCEMENT] Reduce log level verbosity for e2e tests[#3900](https://github.com/grafana/tempo/pull/3900) (@javiermolinar)
-* [ENHANCEMENT] Support reloading client certificates [#537](https://github.com/grafana/dskit/pull/537) (@rubenvp8510)
```

Comment:
Actually, I'm correcting that. Andreas pasted the wrong PR, one from dskit instead of him:

https://github.com/grafana/tempo/pull/3915/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR45